API call

FelixLoether opened the pull request:
Closes #7.

lraivio commented on app/services/api.js:
```diff
@@ -0,0 +1,19 @@
+import toPairs from 'lodash/toPairs';
+
+
+function constructQueryString(params) {
```
Something like [query-string](https://github.com/sindresorhus/query-string) could be used here, but our use cases might be simple enough for it not to be needed.

lraivio commented on app/services/api.spec.js:
```diff
@@ -44,4 +44,28 @@ describe('services/api', () => {
     });
   });
+
+  describe('getUrl', () => {
```
- [ ] This should be 'getUnits'.

FelixLoether commented:
Following changes done:

- Renamed the test to `getUnits`.
- Use query-string.
- Added .npmrc to save exact versions of packages by default when using `--save` or `--save-dev`.

I accidentally rebased the fixup commits myself, but hopefully that shouldn't be a big problem.

lraivio commented:
Nice job!
The .npmrc is a very good addition :+1: It's very easy to accidentally use non-exact package versions otherwise.
